packages/lambda-tiler/src/index.ts

```diff
@@ -20,18 +20,17 @@ import { St } from './util/source.tracer.js';
 
 export const handler = lf.http(LogConfig.get());
 
+/** If the request takes too long, respond with a 408 timeout when there is approx 1 second remaining */
+handler.router.timeoutEarlyMs = 1_000;
```

Comment:
Is 1 second enough for attribution?

Reply:
its `lambda.maxDuration - 1_000` so its 59,000ms since we have a default duration of 1 minute.

Reply:
Haha, I see.